src/App.tsx

```diff
@@ -553,8 +644,45 @@ const MapDisplay = ({map}: { map: Map }) => {
               <img
                 src={piece.piece.getImageDataUrl()}
                 onClick={(event) => {
-                  console.log('clicked', piece.x, piece.y, event.movementX, event.movementY)
-                  debugger
+                  // console.log('clicked', piece.x, piece.y, event.pageX, event.pageY )
+                  const clickX = event.pageX - (piece.x * 100) 
+                  const clickY = event.pageY - (piece.y * 100)
+                  // console.log(clickX, clickY)
+
+                  let octaquadrant = 0
+
+                  if (clickY < 50) {
+                    if (clickX < 50) {
+                      if (clickX < clickY) {
+                        octaquadrant = 3
+                      } else {
+                        octaquadrant = 4
+                      }
+                    } else {
+                      if (((clickX-50) + clickY) < 50) {
```

Comment:
`((clickX-50) + clickY) < 50` and `clickX + (clickY-50) < 50` can both be simplified to `clickX + clickY < 100`. Store it in a variable so it is not typed twice.